src/common/IExtensionSettings.ts

```diff
@@ -0,0 +1,11 @@
+export interface ISettings{
+    batchSize: number;
+    batchMaxTimeout: number;
+    initialBatchSizeLoad: number;
+    logging: ILogging
+}
+
+export interface ILogging {
+    react: boolean,
+    node: boolean,
+    openEdge: string}
```

Comment:
Put curly brace into next line.
Try be consistent with code formatting . Use "format document" for newly created files and "format selection" for your part of code.